desc/plotting.py

```diff
@@ -2574,13 +2577,16 @@ def plot_coils(coils, grid=None, fig=None, return_data=False, **kwargs):
     if grid is None:
         grid = LinearGrid(N=400, endpoint=True)
 
-    def flatten_coils(coilset):
+    def flatten_coils(coilset, check_intersection=check_intersection):
```

Comment:
I think you don't need this here you can just close over the value but it's fine